src/TroykaOLED.h

```diff
@@ -179,34 +76,55 @@ class TroykaOLED {
     uint8_t _i2cAddress;
     uint8_t _width;
     uint8_t _height;
-    uint8_t _numX;
-    uint8_t _numY;
+
+    struct {
+        int16_t x;
+        int16_t y;
+    } _last;
+
     uint8_t _codingName;
     bool _stateInvert;
     bool _stateAutoUpdate;
-    bool _stateImageBG;
-    uint8_t *_bufferDisplay;
+    uint8_t _imageColor;
+
+    union {
+        uint64_t column[HORIZONTAL_SIZE];
+        uint8_t asBytes[(VERTICAL_SIZE >> 3) * HORIZONTAL_SIZE];
+    } _screenBuffer;
+
```

Comment:
I would like get a more detailed comment about this fuction.

Reply:
The same array used as a byte array for hardware exchange, and as a 64bit-words array for native addressing to display columns.